src/templates/export-map.js

```diff
@@ -41,7 +41,7 @@ export const exportMapToHTML = options => {
       <script src="https://unpkg.com/react-virtualized@9.21.0/dist/umd/react-virtualized.js" crossorigin></script>
       
       <!-- Load Kepler.gl latest version-->
-      <script src="https://unpkg.com/kepler.gl/umd/keplergl.min.js" crossorigin></script>
+      <script src="https://unpkg.com/kepler.gl@0.2.4/umd/keplergl.min.js" crossorigin></script>
```

Comment:
same here, why lock to a specific version? This is going to make it hard to manage versions in the future unless we have a deploy script that automatically edits them

Reply:
same story here

Reply:
The exported map should use whatever version the current demo app is on, which should be the latest. You can imagine people use the demo app to create a map and some feature is missing because the exported map is fixed to @0.2.4.